src/redux/book/book.js

```diff
@@ -0,0 +1,25 @@
+// ACTIONS
+const addBook = (book) => ({
+  type: 'ADD__BOOK',
+  item: book,
+});
+addBook();
+
+const removeBook = (id) => ({
+  type: 'REMOVE__BOOK',
+  id,
+});
+removeBook();
+// REDUCERS
+const bookReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'REMOVE__BOOK':
+      return state.filter((book) => book.id !== action.type.id);
+    case 'ADD_BOOK':
+      return [...state, action.item.book];
+    default:
+      return state;
+  }
+};
+
+export default bookReducer;
```

Comment:
- [ ] it would be better to write you Action as in the[ ducks pattern](https://github.com/erikras/ducks-modular-redux) in order to match the requirements 

- [ ] and you should do the same for `Categories`

- Screenshot of the requirements 
<img width="896" alt="Screenshot 2022-11-14 at 15 36 50" src="https://user-images.githubusercontent.com/61790587/201687527-fdba6ddb-2859-4d1b-a664-791ae52714e8.png">
- Action 
<img width="409" alt="Screenshot 2022-11-14 at 15 36 26" src="https://user-images.githubusercontent.com/61790587/201687566-43bd74a3-78e7-4e80-b4ec-2ede02a2aa6f.png">